code-reviews/README.md

```diff
@@ -2,51 +2,48 @@
 
-### Reviewee's guidance
-
-1. When starting to work on a task, break your code into the smallest logically separable units. [TODO: More on size here]
-2. Before submitting a pull request (PR) :
-   * Write tests that cover your changes.
-   * Update the documentation related to your change.
-   * All new and existing tests pass.
-   * The code your wrote follows the style of the project.
-   * Run linting checks.
-   * Do manual testing (run the code) and make sure to not introduce a breaking change.
-
-3. Once done with a unit of code submit a pull request (PR) that follows [pull-request-template.md](pull-request-template.md)
-4. Add one or two reviewers (depending on your project's guidelines). Ideally, you would add at least someone who has expertise and is familiar with the project or the language used and someone who is less familiar with the project or language.
-5. Be open to receive feedback, discuss design/code logic and address all comments as follows:
-   * Resolve the comment if you will make the requested change
-   * Mark the comment as won't fix if you are not going to make the requested changes and explain clearly the reasoning
-   * Ask clarification questions on the review itself (not in a private chat) if you don't understand a reviewer's comment
-   * If a comments thread went beyong 5 or 6 replies and there has been no conclusion, then make direct communication with you reviewer (call them or knock on door)
-   * Mark the comment as pending and clearly state that you are going to address it in a future PR.
-6. If the reviewer's haven't looked at your code in 24-48 hours, ping them to ensure a timely review process.
-7. Once all comments have been addressed and your reviewers have approved you changes, you can submit your code.
+As an author of a PR, you should:
 
-## Reviewer's guidance
+1. Add one or more reviewers (depending on your project's guidelines) to the PR. Ideally, you would add at least someone who has expertise and is familiar with the project or the language used. Adding someone less familiar with the project or the language can aid in verifying the changes are understandable, easy to read, and increases the expertise within the team.
+1. Be open to receive feedback, discuss design/code logic and address all comments as follows:
+   * Resolve a comment, if the requested change has been made.
+   * Mark the comment as "won't fix", if you are not going to make the requested changes and provide a clear reasoning - "I'll do it later" is not an acceptible reason!
+   * If you don't understand a comment, ask questions in the review itself as opposed to a private chat
+   * If a thread gets bloated without a conclusion, have a meeting with the reviewer (call them or knock on door)
+1. If the reviewers have not responded in a reasonable time, ping them or raise the issue in a daily meeting.
```

Comment:
can we keep the definition of reasonable time to be 24 to 48 hours? or whatever the crew decides? It is always good to give an indication for someone starting out and they can always refer to the tech lead or crew to decide on the numbers

Reply:
An example of reasonable time provided.